zbus/src/lib.rs

```diff
@@ -86,7 +86,7 @@ pub use match_rule::*;
 
 mod socket_reader;
 
-mod proxy;
+pub mod proxy;
 pub use proxy::*;
```

Comment:
deprecated export/aliases here too. These changes are supposed to be helping users (not just us devs) and provide them with a Rusty API/patterns.